setup/l10n_hr_base_location/odoo/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
Not wrong, but there is no  need to have the trouble to add these files: they will be automatically generated by an OCA bot.

Reply:
Tnx for pointing out this part... wasn't sure if this will be automatic or not so i added it in first commit, and repeated in this one.. 
Will ommit this in future.